templates/form.twig

```diff
@@ -21,16 +21,19 @@
             <tr>
                 <td class="first"><label for="applyTo">{{ 'CustomAlerts_ApplyTo'|translate }}</label></td>
                 <td>
-
-                    <div piwik-siteselector
-                         class="sites_autocomplete"
-                         siteid="{{ currentSiteId }}"
-                         sitename="{{ currentSiteName }}"
-                         switch-site-on-select="false"
-                         show-all-sites-item="false"
-                         show-selected-site="true"
-                         name="idSite"
-                         id="alertReportSiteSelector"></div>
+                    <div class="sites-select-controls">
+                        <a href="#" id="sites-select-all" class="visible">{{ 'CustomAlerts_SelectAll'|translate }}</a>
+                        <a href="#" id="sites-deselect-all">{{ 'CustomAlerts_DeselectAll'|translate }}</a>
+                        <a href="#" id="sites-inverse-select">{{ 'CustomAlerts_InverseSelection'|translate }}</a>
+                    </div>
+                    {% for site in sites %}
+                        <div class="site-checkbox">
```

Comment:
Maybe in this case let a user select "all websites" by setting `show-all-sites-item="true"`? Or maybe have a button to enable for all websites and when this is enabled, the user can select multiple sites that are excluded from all sites but this will be hard to implement.

How it is currently implemented, "All websites" does not work anyway. When selecting "All websites" a user would expect that also newly created websites will receive this alert. In the current implementation this would only affect currently existing websites. For this we'd need to save the string `all` as idSite but this requires an implementation change. Also if you then want to exclude sites from `all` it'll become more complicated as you will need to store these sites separately. Maybe a new column in `alert` table like `all_sites=1/0` could help and depending on this value the referenced idSites are added to this alert or extracted from all websites.

Even with just 50 websites, which is quite common, this UI is already not really practical anymore as it would show 50 checkboxes for all the sites. Ideally we find a solution that works good for all cases.